src/afl-fuzz-run.c

```diff
@@ -504,6 +505,9 @@ u8 calibrate_case(afl_state_t *afl, struct queue_entry *q, u8 *use_mem,
 
     fault = fuzz_run_target(afl, &afl->fsrv, use_tmout);
 
+    // update the time spend in calibration after each execution, as those may be slow
+    update_calibration_time(afl, &calibration_start_us);
```

Comment:
why would you update every iteration? Also, you update with the calibration_start_us difference every single time, this does not work out well :)